docs/GraFx-Studio/convert/Adobe-InDesign/index.md

```diff
@@ -0,0 +1,196 @@
+| **Image Frames**       | Image frame with arbitrary position      | ✅             |                                                    |
+|                        | Image frame with rotated content         | ✅             |                                                    |
+|                        | Image frame with stretched content       | ✅             |                                                    |
+| **Primitives**         | Rectangle, ellipse, triangle             | ✅             |                                                    |
+|                        | Stroke weight                            | ✅             |                                                    |
+|                        | Stroke color (predefined and custom)     | ✅             |                                                    |
+|                        | Fill color (predefined and custom)       | ✅             |                                                    |
+|                        | Rectangle corner radius                  | ✅             |                                                    |
+|                        | Triangle corner radius                   | ❌ |                                    |
+|                        | Polygon stroke weight                    | ❌  |                                    |
+| **Predefined Color Export** | CMYK swatch colors| ✅           |                                                    |
+|  | RGB swatch colors| ✅           |                                                    |
+|  | Spot color swatch colors| ✅           |                                                    |
+| **Preflight**          | List of potential problems               | ✅             |                                                    |
+|                        | Details (click on an item in the list)   | ✅             |                                                    |
+|                        | Center problem object (click on warning) | ✅             |                                                    |
+|                        | No document open behavior                | ✅             | Preflight does nothing if no document is open     |
+|                        | Switching documents                      | ✅             | Preflight results reset when switching documents  |
+| **Layers**             | Layer names                              | ✅             | InDesign layer names converted to default names   |
+|                        | Hidden layers                            | ❌           | Hidden layers are not exported                    |
+
+### Legend (not John)
+
+- ✅ **Supported**: Fully supported
+- ⚠️ **Limited Support**: Partially supported; some functionality may be restricted
+- ❌ **Unsupported**: The team is working hard to add this functionality
+
+## Tips for Successful Conversion
+
+1. **Simplify Your Design**:
```

Comment:
This numbered list doesn't work out as intended. Please check the staging site and you will see what I mean.